### PR TITLE
Update new-hello-world for 0.41

### PR DESCRIPTION
**Pull request by tylerbutler**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,44 +18,90 @@ npm start
...
+groupings as well as provide an API surface to your data.
 
-The KVPair DataObject provides the basic functionality to set and get data of any type on a given key, and inform your application when the data in that key changes.
+In subsequent demos we will look at building custom DataObjects, but for many use cases (including this dice roller) the
+built in distributed data structures will work perfectly.
+
+The SharedMap distributed data structure provides the basic functionality to set and get data of any JSON-serializable
+type on a given key, and inform your application when the data in that key changes.
 
 #### Using KeyValuePair DataObject
```
Comment (skylerjokiel) [inline README.md:45]:
```suggestion
#### Using SharedMap Distributed Data Structure
```

File: README.md [hunk 2]
```diff
@@ -18,44 +18,90 @@ npm start
...
 
-The KVPair DataObject provides the basic functionality to set and get data of any type on a given key, and inform your application when the data in that key changes.
+In subsequent demos we will look at building custom DataObjects, but for many use cases (including this dice roller) the
+built in distributed data structures will work perfectly.
 
-#### Using KeyValuePair DataObject
+The SharedMap distributed data structure provides the basic functionality to set and get data of any JSON-serializable
+type on a given key, and inform your application when the data in that key changes.
+
+#### Using SharedMap Distributed Data Structure
```
Comment (tylerbutler) [inline README.md:45]:
```suggestion
#### Using SharedMap Distributed Data Structure

The SharedMap distributed data structure provides the basic functionality to set and get data of any JSON-serializable
type on a given key, and inform your application when the data in that key changes.

```

**Review by micahgodbolt — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -18,44 +18,91 @@ npm start
...
 
 // Changed callback is called each time a key's value is changed
-kvpair.set('MyDataKey', 'some new data'); // logs { key: 'MyDataKey', path: '/', previousValue: 'some data' }
+myMap.set('MyDataKey', 'some new data'); // logs { key: 'MyDataKey', path: '/', previousValue: 'some data' }
 ```
 
 ### Creating a Fluid Container
 
-Fluid [Containers](https://fluidframework.com/docs/glossary/#container) are your applications entry point to the Fluid Framework.
+[Fluid containers](https://fluidframework.com/docs/glossary/#container) are your application's entry point to the Fluid
```
Comment (sumedhb1995) [inline README.md:64]: nit: Would it be confusing for devs reading this to be pointed to documentation for the runtime-level `Container`, when TinylicousClient will give them back a `FluidContainer`? Terminology wise, they may be wondering why the two objects to match.

File: README.md [hunk 4]
```diff
@@ -18,44 +18,91 @@ npm start
...
+`TinyliciousClient` needs to be initialized before use and can take an optional configuration object to changes settings
+such as default port.
+
+### Accessing Fluid Data
 
-To create our container we combine a unique ID (pulled from a hash in the URL) with the container factory that includes DataObject we want to use. Since these containers are created on the client, the client needs to determine if the document is existing or new, so in the absence of that URL hash, we'll pass in `true` for the 3rd parameter, `createNew`.
+Before we can access any Fluid data, we need to make a call to the `TinyliciousClient` with necessary service
+configuration and container schema.
+
+- `serviceConfig` is will vary depending on the service. `TinyliciousClient` only requires an `id` for the container.
```
Comment (sumedhb1995) [inline README.md:85]:
```suggestion
- `serviceConfig` will vary depending on the service. `TinyliciousClient` only requires an `id` for the container.
```

File: src/app.ts [hunk 5]
```diff
@@ -3,30 +3,35 @@
...
+        : await TinyliciousClient.getContainer(serviceConfig, containerSchema);
 
     renderView(
-        fluidContainer.initialObjects.dice as ISharedMap, 
+        fluidContainer.initialObjects.dice as ISharedMap,
         document.getElementById('content') as HTMLDivElement
     );
+
+    // returned initialObjects are live Fluid data structures
+    return fluidContainer.initialObjects;
```
Comment (sumedhb1995) [inline src/app.ts:34]: nit: unnecessary return?

**Review by sumedhb1995 — APPROVED** [review]
🚢 Looks great, just a few small nit suggestions! Thank you for documenting this!

File: src/app.ts (hunk 5 above)
Comment (tylerbutler) [inline src/app.ts:34]:
```suggestion
```

**Pull request merged**